gnssanalysis/gn_io/sp3.py

```diff
@@ -477,28 +477,28 @@ def getVelPoly(sp3Df: _pd.DataFrame, deg: int = 35) -> _pd.DataFrame:
 
     """
     est = sp3Df.unstack(1).EST[["X", "Y", "Z"]]
-    x = est.index.get_level_values("J2000").values
-    # x = est.index.values
-    y = est.values
+    time_values = est.index.get_level_values("J2000").values
+    xyz_positions = est.values
 
-    off, scl = mapparm([x.min(), x.max()], [-1, 1])  # map from input scale to [-1,1]
+    # map from input scale to [-1,1]
+    offset, scale_factor = mapparm([time_values.min(), time_values.max()], [-1, 1])
```

Comment:
not sure we need `factor` scale can be enough

Reply:
Just noticed it was `scale_factor` in some other similar code, which got introduced during a PR review:

https://github.com/GeoscienceAustralia/gnssanalysis/commit/fe81af2bef28727413acccf89b8c43a778d33aea#diff-330d45d4cfeafdbd9dcc218de027fe2763d140a22cf9763c6cf944f9ad0f1543R108